doc/ci_admin_guide/Update_private_docker_registry.md

```diff
@@ -0,0 +1,52 @@
+# How to update private docker registry
+
+## When is the update needed? What need to be updated?
+
+The update of local docker registry is required when the version of Controller used in Windows CI is updated to a newer one. All Contrail images required by Contrail Ansible Deployer running in Windows CI have to be updated to version matching selected Controller version. Some new cached Openstack images may be required by updated Contrail Ansible Deployer.
+
+## How to update
+
+### How to update Contrail images
+
+1. Choose Contrail version tag you want to be cached in private docker registry. All available tags can be found [here](https://hub.docker.com/r/opencontrailnightly/contrail-vrouter-agent/tags/).
+2. Log into `winci-registry` VM as root. Check if there is `update-docker-registry.py` script in root directory.
```

Comment:
`Check if there...` - let's put a step before about entering a specific directory and checking if script exist there.

Reply:
I would put `Check if ..` in a separate step. Then `If there is no such script` would be its substep.

I would also add the `curl`/`wget` command to download a script.

Reply:
`Log into winci-registry` - let's change this to:

```
Log into winci-registry:
    - Please contact Windows CI team for access
```